db/schema_tables.cc

```diff
@@ -1697,6 +1697,13 @@ static std::vector<data_value> read_arg_values(const query::result_set_row& row)
     }
 #endif
 
+static std::pair<cql3::functions::function_name, std::vector<data_type>> drop_func(replica::database& db, const query::result_set_row& row) {
+    cql3::functions::function_name name{
+            row.get_nonnull<sstring>("keyspace_name"), row.get_nonnull<sstring>("function_name")};
+    auto arg_types = read_arg_types(db, row, name.keyspace);
+    return std::make_pair(std::move(name), std::move(arg_types));
+}
```

Comment:
The function is called drop, but it doesn't drop anything.

Reply:
The drop function is dropped in the rebase